### ISSUE TITLE
apply: Creating pipe: Too many open files

### ISSUE THREAD

**Issue by stardiviner** [ofosos/org-epub#26]
I have a large Org file, when exporting it to make a EPUB ebook, I got a warning:
```
apply: Creating pipe: Too many open files
```
Can it be fixed or improved?

org version: 9.1.14
emacs version: 26.1
platform: Arch Linux, 4.18.16
installed via: MELPA

**Comment by ofosos** [comment]
I looked into this but still can't understand it. Will need some more time.

**Comment by stardiviner** [comment]
My Org file is very big, it has 33421 lines. And many source blocks and links, maybe this caused error.

**Comment by ofosos** [comment]
My understanding is that it should not do that.

**Comment by stardiviner** [comment]
I don't know how to debug this issue, can you reproduce this problem and debug it?

**Comment by ofosos** [comment]
@stardiviner are you still facing this issue?

**Comment by stardiviner** [comment]
Yes. A really big Org file has lot of link and fontify and source blocks has this issue still.
